yt/frontends/chombo/data_structures.py

```diff
@@ -633,6 +640,7 @@ def _read_particles(self):
 
 
 class Orion2Dataset(ChomboDataset):
+    _load_requirements = ["h5py"]
```

Comment:
I had a preference for overloading the base class' attribute when the child class _also_ had requirements, even if they are similar. It's not exactly that I wanted to be explicit, but rather that I'm trying to avoid entangling classes unnecessarily.